src/components/learner-credit-management/cards/CourseCard.jsx

```diff
@@ -72,6 +75,8 @@ const CourseCard = ({
 
   const isExecEd = courseType === EXEC_ED_COURSE_TYPE;
 
+  const linkToCourse = `${ENTERPRISE_LEARNER_PORTAL_URL}/${enterpriseSlug}/course/${key}`;
```

Comment:
While this will technically work for both Open Courses and Executive Education courses alike, I might recommend conditionally using the Executive Education specific page route rather than relying on the Open Courses page route to detect the course is Executive Education and do an automated redirect.

Given we have the `isExecEd` boolean above, we could have `linkToCourse` conditional based on course type:

```
${ENTERPRISE_LEARNER_PORTAL_URL}/${enterpriseSlug}/executive-education-2u/course/${key}
```

This would ensure the GetSmarter logo is shown in the header immediately on page load instead of flickering in after we retrieve course metadata to identify the course type.

<img width="1183" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/7fac7859-4509-434d-b995-b837d7777b94">